Traceback (most recent call last):
  File "c:\Users\bcat4\VSCPythonProjects\exercisesATBSWP\errorexample.py", line 4, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
